content/events/ftc-at-ethcc.md

```diff
@@ -1,11 +1,18 @@
 ---
 eventName: FTC at EthCC
+eventDescription: "Funding the Commons is focused on new models of sustainable public goods funding and value alignment in open source networks. We bridge the public goods community across Web2, Web3, research, philanthropy and industry. We do this by convening builders, academics, and funders in both traditional conference and "open space" formats."
+event type: "Conference"
 location: "Paris, France"
-website: 
+venueName: "Sorbonne University"
+venueAddress: "15-21 Rue de l'École de Médecine"
+venueMapsLink: "https://goo.gl/maps/d2fYuBMNrYmi7sQo7"
+website: "https://fundingthecommons.io/paris-2023"
 startDate: 07/15/2023
 endDate: 07/16/2023
 tag: PLN Event
 dateTBD: false
+preferredContacts:
+- 'Twitter|https://twitter.com/fundingcommons'
```

Comment:
@laeccles Please follow the same spacing format just like the eventHosts attribute - 

preferredContacts:
  - 'Twitter|https://twitter.com/fundingcommons'

Reply:
@WinsCodes I have updated this